src/views/studio/EditVideoSheet/EditVideoTabsBar/EditVideoTabsBar.style.ts

```diff
@@ -19,10 +23,12 @@ export const TabsContainer = styled.div`
   display: flex;
   align-items: center;
   overflow: auto hidden;
+  width: 100%;
+  &::-webkit-scrollbar {
+    height: 0;
+  }
```

Comment:
This should hide the scrollbar on chrome, right? What about firefox? After quick research I think we could add experimental `scrollbar-width: none;` I think I used it also in the regular `Tabs` component.